atom/browser/api/atom_api_session.cc

```diff
@@ -692,7 +692,7 @@ void Session::CreateInterruptedDownload(const mate::Dictionary& options) {
   }
   auto* download_manager =
       content::BrowserContext::GetDownloadManager(browser_context());
-  download_manager->GetDelegate()->GetNextId(base::Bind(
+  download_manager->GetDelegate()->GetNextId(base::BindRepeating(
```

Comment:
```suggestion
  download_manager->GetDelegate()->GetNextId(base::BindOnce(
```

https://cs.chromium.org/chromium/src/content/public/browser/download_manager.h?l=231&rcl=8d3660a6b51f5ca40d1ce56981ec107645344ca0

Reply:
@nornagon i originally thought this too, but that's not in fact the callback being called: it's `GetNextID` on the _delegate_ which is a `RepeatingCallback` https://cs.chromium.org/chromium/src/content/public/browser/download_manager_delegate.h?sq=package:chromium&g=0&l=73